src/ontology/uberon-edit.obo

```diff
@@ -7007,8 +7007,8 @@ consider: FBbt:00007012
 consider: FMA:86454
 consider: GO:0044464
 consider: HAO:0000014
+consider: http://purl.obolibrary.org/obo/NIF_Subcellular_sao-1337158144
```

Comment:
Sorry for not catching this before, but this CURIE should not be changed to a full link using OBO purl.

Reply:
Oh, thanks for catching it!  I am not sure how that happened, though, it was not something I did purposefully. 

Do you have any suggestions on how I should fix it? Is there something Protégé-based that I should do?

Reply:
Probably add an appropriate `idspace` declaration to the OBO file? This is a new feature in the newest OWL API.

Reply:
@lubianat, to clarify, which Protégé version do you use?

Reply:
@anitacaron not anymore... big change.

Reply:
Originally: https://github.com/owlcs/owlapi/pull/1072

Reply:
do you know the prefix? 😬

Reply:
GO is using `http://www.neurolex.org/wiki/` https://github.com/geneontology/go-site/blob/f20ccace8dc1fdd95b0267cd1f4015b433f8cb91/metadata/db-xrefs.yaml#L1729

Reply:
So try adding `idspace: NIF_Subcellular http://www.neurolex.org/wiki/` in the file header.

Reply:
ok, it worked! Thanks, @balhoff!